src/Sql/Sql/Database Backup/Cmdlet/AzureSqlDatabaseBackupShortTermRetentionPolicyCmdletBase.cs

```diff
@@ -0,0 +1,160 @@
+﻿using System;
```

Comment:
@dealaus it looks like the license header is missing from a few of the `.cs` files in this PR -- please make sure it's added to those files